app/src/main/java/com/iitp/njack/iitp_connect/common/di/modules/android/ActivityBuildersModule.java

```diff
@@ -17,4 +20,13 @@
 
     @ContributesAndroidInjector
     abstract HomeActivity contributeHomeActivity();
+
+    @ContributesAndroidInjector
+    abstract KickoffActivity contributeKickoffActivity();
+
+    @ContributesAndroidInjector
+    abstract EmailActivity contributeEmailActivity();
+
+    @ContributesAndroidInjector
+    abstract CredentialSaveActivity contributeCredentialSaveActivity();
```

Comment:
I don't see these activities defined in the code? Are they automatically created by the Firebase SDK?

Reply:
The activities are part of firebaseUI library.

Reply:
They are not automatically created,but app could not run without injecting them.